### ISSUE TITLE
Improvements for New My Tracks

### ISSUE THREAD

**Issue by pebogufi** [osmandapp/OsmAnd#17539]
I would like to ask for two improvements for the new design of My Tracks.

+ on renaming a filename please don't change file datetime.

+ please add a function to refresh / reload the filelist. 
I often make changes on subfolders of tracks with a external filemanager (Total Commander). Currently i have to leave My Tracks completely to get the list refreshed. In a lot of tools a swipe down in the list window is used to perform a refresh, i would recommend that too for My Tracks (or a menu item).

**Comment by DmitryAlexei** [comment]
- As for the first point, it is reproduced

<details><summary>Track creation date changes</summary>
<video src="https://github.com/osmandapp/OsmAnd/assets/116079541/1ecf3a8d-abea-488c-971a-8eb32e1648a1"></video>
</details>

- On the second point. Only a few users may need this feature, since Android in the latest versions of the system has significantly limited the ability to work with system files.

**Comment by sonora** [comment]
Regarding (2): For me it would suffice to briefly change the folder and back, like most file managers would update the view then - no need for an explicit "refresh" button.

General remark regarding a "refresh": I have ~4000 tracks in ~70 or so sub-folders, and experience often long wait times to enter the new Tracks screens (both the management one and then again the "Visible on map" variant). The times vary quite a bit from attempt to attempt, between 2 and 5 seconds - I am not sure why this is, and what is being done in those periods. Probably we have to put some logging and check ...

**Comment by pebogufi** [comment]
> Regarding (2): For me it would suffice to briefly change the folder and back, like most file managers would update the view then - no need for an explicit "refresh" button.
> 
Agree. Currently I have to leave My Tracks complete. Leaving only one level back would be ok for me.

**Comment by pebogufi** [comment]
> General remark regarding a "refresh": 
Time to open My Tracks for me
Device 1: Samsung tab S5e Android 10, files in SD card, obb - less than 1 sec, very good
Device 2: Samsung S10 Android 12, files in internal, media, - about 5 sec.
The file structure, no of files, size is nearly identical. Normaly Device 2 is MUCH faster. 
This result is quite funny...

**Comment by pebogufi** [comment]
Well done, thanks
